--- ./build/cmake/CMakeLists.txt
+++ ./build/cmake/CMakeLists.txt
@@ -125,6 +125,9 @@
   target_link_libraries (glew LINK_PRIVATE -nodefaultlib -noentry)
   target_link_libraries (glew LINK_PRIVATE libvcruntime.lib)
   target_link_libraries (glew LINK_PRIVATE msvcrt.lib )
+  if(CMAKE_SYSTEM_PROCESSOR MATCHES "^(ARM64|arm64|aarch64)$")
+    target_link_libraries (glew LINK_PRIVATE ucrt.lib)
+  endif()
   string(REGEX REPLACE "/RTC(su|[1su])" "" CMAKE_C_FLAGS_DEBUG ${CMAKE_C_FLAGS_DEBUG})
 elseif (WIN32 AND ((CMAKE_C_COMPILER_ID MATCHES "GNU") OR (CMAKE_C_COMPILER_ID MATCHES "Clang")))
   # remove stdlib dependency on windows with GCC and Clang (for similar reasons
